venia-integration-tests/cypress.config.json

```diff
@@ -0,0 +1,12 @@
+{
+    "baseUrl": "https://develop.pwa-venia.com/",
+    "defaultCommandTimeout": 10000,
+    "env": {},
+    "fixturesFolder": "src/fixtures",
+    "integrationFolder": "src/tests",
+    "pageLoadTimeout": 30000,
+    "requestTimeout": 30000,
+    "testFiles": "**/*.spec.js",
```

Comment:
May want to consider restricting this to the `src/tests` folder, but probably fine

Reply:
Yes, it checks for *.spec.js in the `integrationFolder`. So basically we are checking for those files in the `src/tests` folder.